### PR TITLE
Added code for new itp package bug - (Solves - #55)

### LINKED ISSUE #55: Lintian - Package should close ITP bug
Lintian reports the issues:

W: plymouth-theme-meilix-text: new-package-should-close-itp-bug
W: plymouth-theme-meilix-logo: new-package-should-close-itp-bug

See: https://lintian.debian.org/tags/new-package-should-close-itp-bug.html

### PR DESCRIPTION
**Pull request by pranav1698**
Fixes #55 
 - Short description of what this resolves:
   The new package error does not appear now

- Changes proposed in this pull request:
   Changelog is edited for solving the new package bug

### CODE REVIEW

File: meilix-artwork/debian/changelog [hunk 1]
```diff
@@ -3,3 +3,9 @@ meilix-artwork (1.0-1) unstable; urgency=low
   * Prepare for Meilix
 
  -- Meilix <mb@mariobehling.de>  Thu, 28 Mar 2013 16:04:48 +0700
+
+meilix-artwork (1.0-1) unstable; urgency=low
+
+  * Closes #nnnnn
```
Comment (tabesin) [inline meilix-artwork/debian/changelog]: #nnnnn is a dummy.

**Comment by pranav1698** [comment]
@tabesin review now

**Comment by pranav1698** [comment]
The bug number should be random or chronological like the pr
Could you help me on how to adding name as a release manger, I saw on xubuntu-artwork, the format they use is pretty much the same?

**Comment by tabesin** [comment]
Just don't write Mario as previously in your PR but your name ;)

**Pull request merged**